app/configuration/Config.scala

```diff
@@ -22,4 +23,5 @@ object Config {
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$contributeUrl$uri") & idSkipConfirmation
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
 
+  val guardianDomain = ".theguardian.com"
```

Comment:
Are you referring to `Config.domain`? This is derived from `contribute.url` which on production is set to `contribute.theguardian.com`, so I don't think we could use that.